README.md

```diff
@@ -10,10 +10,12 @@ Pie chart for React Native, works on **iOS** and **Android** both
 ## Install
 
 `npm i --save react-native-pie`
+`yarn add react-native-pie`
```

Comment:
add `or`? if would be confusing for someone who don't familiar with yarn

Reply:
https://github.com/nihgwu/stickyard#install

Reply:
Sure I'll address this